packages/components/Button/src/Button.styling.ts

```diff
@@ -98,17 +100,26 @@ export const getDefaultSize = (): ButtonSize => {
 
 export const getPlatformSpecificAppearance = (appearance: ButtonAppearance): ButtonAppearance => {
   switch (appearance) {
-    case 'ascent': // Included to cover Mobile platform naming guidelines, maps to 'primary'.
+    case 'accent': // Included to cover Mobile platform naming guidelines, maps to 'primary'.
       return 'primary';
+
     case 'outline': // Exists only for Mobile platforms, maps to default on other platforms.
-      if (Platform.OS == 'android' || Platform.OS == 'ios') return 'outline';
-      else return null;
+      if (Platform.OS == 'android' || Platform.OS == 'ios') {
```

Comment:
nit: Could you pull out this platform check into a meaningful variable like "doesSupportOutline" or something like that? It makes it easier to understand the intent of the platform check, esp if something should be changed in the future like adding another platform to the check.

Same comment for the default case below